oio/cli/admin/common.py

```diff
@@ -14,12 +14,30 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 
-class AccountCommandMixin(object):
+class ItemCheckCommandMixin(object):
     """
-    Add account-related argmuments to a cliff command.
+    Add item check related arguments to a cliff command"
     """
 
     def patch_parser(self, parser):
+        parser.add_argument(
+            '--depth', '--max-depth',
+            type=int,
+            default=5,
+            help=("How deep to recurse. 0 means do not recurse. "
+                  "N > 0 means recurse N levels below the specified item type "
+                  "(namespace -> account -> container -> object -> chunk, "
+                  "default: 5).")
+        )
+
+
+class AccountCommandMixin(ItemCheckCommandMixin):
```

Comment:
For what reason do we need to inherit from `ItemCheckCommandMixin`. I don't get it.